tls/s2n_config.c

```diff
@@ -1219,3 +1219,14 @@ int s2n_config_get_supported_groups(struct s2n_config *config, uint16_t *groups,
 
     return S2N_SUCCESS;
 }
+
+int s2n_config_set_serialized_connection_version(struct s2n_config *config, s2n_serialized_connection_version version)
+{
+    POSIX_ENSURE_REF(config);
+
+    /* Currently there is only one format version supported */
+    POSIX_ENSURE_EQ(version, S2N_SERIALIZED_CONN_V1);
```

Comment:
Should we make sure that the handshake isn't complete / hasn't started? Otherwise you could probably circumvent this, including accidentally. Just set the version right before you call serialize :)

Reply:
I don't think you can do this type of check, at least not in this function? Because this is a config method and not a connection method.

Reply:
I guess we already say not to swap the config out during a connection. I'm probably just being paranoid.